perfkitbenchmarker/publisher.py

```diff
@@ -18,7 +18,7 @@
 
 import abc
 import collections
-import copy
+from copy import copy, deepcopy
```

Comment:
Use import for modules only. See this style guide that we try to follow: https://google.github.io/styleguide/pyguide.html#Imports

Reply:
Updated https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/1551/commits/b3edfe0eba294ec506f773b9a513fd353e0c3fcf